geomesa-web/geomesa-web-csv/pom.xml

```diff
@@ -15,29 +15,93 @@
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-web-core-accumulo1.5</artifactId>
-            <version>1.0.0-rc.4-SNAPSHOT</version>
+            <version>${project.parent.version}</version>
```

Comment:
I think it is like that in most places already... it probably isn't worth changing wholesale, we can address it on a case-by-case basis.